services/lgtm/lgtm-base.js

```diff
@@ -25,8 +31,17 @@ module.exports = class LgtmBaseService extends BaseJsonService {
     return { label: 'lgtm' }
   }
 
-  async fetch({ user, repo }) {
-    const url = `https://lgtm.com/api/v0.1/project/g/${user}/${repo}/details`
+  static get pattern() {
+    return ':host/:user/:repo'
+  }
+
+  static get examplePattern() {
```

Comment:
Once the pattern is made more explicit as noted above, this (and usage of it) should be deleted as the examples will be able to use the default route pattern defined for the services